torch/_meta_registrations.py

```diff
@@ -239,6 +239,11 @@ def meta_bernoulli_(self, p=0.5, generator=None):
     return self
 
 
+@register_meta(aten.bernoulli.p)
+def meta_bernoulli_p(self, p, generator=None):
+    return torch.empty_like(self)
```

Comment:
Why is one empty_like and the other new_empty? haha

Reply:
I guess in this case it doesn't matter :p ill make it consistent